Tests/KituraWebSocketTests/WebSocketClient.swift

```diff
@@ -431,7 +519,7 @@ class WebSocketMessageHandler: ChannelInboundHandler, RemovableChannelHandler {
 class HTTPClientHandler: ChannelInboundHandler, RemovableChannelHandler {
 
     typealias InboundIn = HTTPClientResponsePart
-    unowned var client: WebSocketClient
```

Comment:
@harish1992 Are you sure we don't need this to be an `unowned` reference any more?

Reply:
The reference is required. Thanks for pointing out.